### DISCUSSION TITLE
A few ideas

### DISCUSSION THREAD

**Discussion by ghost** [Keriew/augustus#251] [Ideas]
1. Autosave system.
Its almost a useless thing when you have in-game problem (arent ready for current battle, arent ready to send emperor's order, etc), and, as you know, all new "problems" (messages etc) we got on start of month, and you try to load autosave file to do some backtime (atleast for a little time to get ready)... and f*ck, it was saved exacly second a second with that message. Cause my "engrish" i cant translate all my ANGRY about this, these autosaves is most useless thing in Julius. As any old fan of c3, i always try to play next campaign with some "experiments", some kind of new type of playing, with my own "quests" and achievements ( like GamerZakh played on youtube), to do every campaign a little different and interesting. But its SO annoying  to have autosave system, but never use cause useless. tell me... what is benefit to have a autosave only one month old? I BEGGIN YOU, please, do something with it. Couple of ideas about this i type here -> https://github.com/bvschaik/julius/issues/246

I hope... very hope...

2. Some idea to facilitate building process:
Oftenly we build some amazing city with rush and entusiasm but these trees and rocks stops us and we need to choose a shovel, clear another piece of trash, and it repeats a many times while playing. So, what u say if we have some options, like:
2.1. Choose School > Click on empty land > School is builded. (classic, this we have already)
2.2. Choose School > CTRL + Click on land with some destroyable trash (trees etc) > Trash automaticly cleared (with own cost for clearing) and School is builded. This method could be as an option in Augustus configuration menu, but i think it is VERY comfortable to biuld a lot in one run during paused (every one loves these runs xD ). As GamerZakh sayd: "its not about difficulity, its about annoying button clicking", when we have to oftenly swap to shovel cause one tree interferes to build a whole Coliseum.

THANKS A LOT! Augustus Forever <3

**Comment by Keriew** [comment]
I agree, I like the ideas. I'll get to implementing them after cleaning up the issues involved with the current version.

**Comment by ghost** [comment]
thanks a lot!

**Comment by crudelios** [comment]
After speaking with @bvschaik, apparently the game has a native option to auto-clear, but it's untested. There even is a string for "auto-clear on/off". If it works we can add it to Julius, otherwise we'll roll out our own implementation.

In short: auto-clear will eventually be available on base Julius :)

Regarding autosave: I created an autosave branch some time ago, I'll adapt it to Augustus.

**Comment by cdsmith-umn** [comment]
I would like to add a request. If possible, make the ratings (culture, prosperity, peace, favor) increase in the campaign missions on harder difficulty settings. I decided to check to see which difficulty I was playing on (hard) and have changed it to "very hard". I was disappointed to see that little changes between the difficulties. I don't really know by how much they should increase. I was thinking maybe 30% on "very hard", as compared to what they currently are - but this could be too much.

Alternatively, perhaps this could be made into a configuration option. If so, then perhaps it could be something that a player could set, making the difficulty customisable.

**Comment by Areldir** [comment]
Can this be moved to the Discussions tab too? If so, I'll close the issue.
